assets/css/dna.scss

```diff
@@ -0,0 +1,50 @@
+$default-font: "Roboto", "Times New Roman", serif;
+$second-font: "Raleway", "Times New Roman", serif;
+
+$light-green: #497557;
+$green: #0c4a34;
+$grey: #757575;
+$dark-green: #02291c;
+$text-color-dark: #214e41;
+$text-color-light: #43806c;
+$white: #ffffff;
+
+$mobile: 375px;
```

Comment:
$mobile: 375px;
$tablet: 768px;
$desktop: 1280px;

Variables are declared in multiple files. The correct approach is to declare only in one file. Then import to other

Reply:
this is about every variables - hold them and imort from one special file